backend/app/api/endpoints/openapi_responses.py

```
@@ -324,6 +344,249 @@ async def create_response(
+        execution_request = await build_execution_request(
+            task=setup.task,
+            assistant_subtask=setup.assistant_subtask,
+            team=team,
+            user=user,
+            message=input_text,
+            enable_tools=enable_chat_bot,
+            enable_deep_thinking=enable_chat_bot,
+            enable_web_search=enable_chat_bot and settings.WEB_SEARCH_ENABLED,
+            preload_skills=preload_skills,
+        )
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)